README.md

```diff
@@ -59,55 +59,57 @@ Time to set up the server that will run Cartography.  Cartography _should_ work
 
 			⚠️ At this time we run our automated tests on Neo4j version 3.5.\*.  Other versions may work but are not explicitly supported. ⚠️
 
-	2. [Install](https://neo4j.com/docs/operations-manual/current/installation/) Neo4j on the server you will run Cartography on.
+	1. [Install](https://neo4j.com/docs/operations-manual/current/installation/) Neo4j on the server you will run Cartography on.
```

Comment:
Numbering on readme is broken now

Reply:
This is the standard way to doing numbering in markdown. It will render the page the same as it was, with incrementing numbers, but it will be easier to modify in the future.